packages/terra-action-header/package.json

```diff
@@ -29,6 +29,7 @@
     "prop-types": "^15.5.8",
     "terra-button": "^3.17.0",
     "terra-button-group": "^3.17.0",
+    "terra-collapsible-menu-view": "^6.5.0",
```

Comment:
I'm conflicted about adding a dependency for just an example.

Reply:
Yeah, the odd part about it is with terra-collapsible-menu-view being it terra-framework, it will make major version bumps weird when releasing terra-collapsible-menu-view.

I'm fine with it being a dependency if we move terra-action-header to terra-framework.

Reply:
Let's just document rather than adding this dependency just to have an example

Reply:
Done in 380ceaf77a0f7641c18c8b17327a3849634461a9